src/c++/vdo/base/data-vio.c

```diff
@@ -1320,12 +1321,14 @@ static void clean_hash_lock(struct vdo_completion *completion)
 static void finish_cleanup(struct data_vio *data_vio)
 {
 	struct vdo_completion *completion = &data_vio->vio.completion;
+	u32 discard_size = min_t(u32, data_vio->remaining_discard,
```

Comment:
Looks like there are two uses of VDO_BLOCK_SIZE - data_vio->offset. Might be good to make one local variable that explains in the name its purpose.